v2/holiday.go

```diff
@@ -42,13 +42,14 @@ type Holiday struct {
 	Except      []int          // years where the holiday doesn't apply
 
 	// calculation fields; required fields depend on rule being followed
-	Month    time.Month   // the month the holiday occurs
-	Day      int          // the day the holiday occurs
-	Weekday  time.Weekday // the weekday the holiday occurs
-	Offset   int          // the weekday or start date offset the holiday occurs
-	Julian   bool         // the holiday is based on a Julian calendar
-	Observed []AltDay     // the substitution days for the holiday
-	Func     HolidayFn    // logic used to determine occurrences
+	Month      time.Month   // the month the holiday occurs
+	Day        int          // the day the holiday occurs
+	Weekday    time.Weekday // the weekday the holiday occurs
+	Offset     int          // the weekday or start date offset the holiday occurs
+	OffsetDays int          // days offset from the date the holiday would occur, used for DayAfter Holidays
```

Comment:
I don't want to bikeshed this too much, but I think `OffsetDays` is too close to `Offset` and may be confusing. Can you make it something like `AdjustDays` or `CalcOffset` or something like that to make it clear that the offset is applied after the calculation?

Reply:
Changed to `CalcOffset`